doc/news/_preparation_next_release.md

```diff
@@ -48,7 +48,9 @@ docker run -it elektra/elektra
 - <<HIGHLIGHT>>
 - <<HIGHLIGHT>>
 
-### <<HIGHLIGHT>>
+### New Backend
```

Comment:
@kodebach please write something here and link to the main documents describing new-backend. This is also important for internal development that people know that there was a huge change.

Reply:
True, the commands should all (or almost all) work. I'll uncomment it again and add a warning that some of the explanation around the commands might be outdated.